.tekton/tektoncd-hub-main-hub-ui-push.yaml

```diff
@@ -30,12 +30,6 @@ spec:
     value: quay.io/redhat-user-workloads/tekton-ecosystem-tenant/tektoncd-hub-main/hub-ui:{{revision}}
   - name: dockerfile
     value: openshift/dockerfiles/hub-ui.Dockerfile
-  - name: build-platforms
```

Comment:
Why does it remove this ? 🤔

Reply:
Its because we added default https://github.com/openshift-pipelines/hack/blob/main/cmd/konflux/templates/tekton/docker-build-ta.yaml#L11C1-L16C19 for all the platform

Reply:
ah right !